x-pack/plugins/security_solution/public/common/components/drag_and_drop/draggable_wrapper.tsx

```diff
@@ -131,288 +128,212 @@ export const getStyle = (
+              renderClone={RenderClone}
+            >
+              {DroppableContent}
+            </Droppable>
+          </DragDropErrorBoundary>
+        </Wrapper>
+      ),
+      [DroppableContent, RenderClone, dataProvider.id, isDisabled]
+    );
+
+    if (isDisabled) return <>{content}</>;
     return (
-      <WithHoverActions
-        alwaysShow={isShowingTopN || hoverActionsOwnFocus}
-        closePopOverTrigger={closePopOverTrigger}
-        hoverContent={disableHoverActions(scopeId) ? undefined : hoverContent}
-        onCloseRequested={onCloseRequested}
-        render={renderContent}
-      />
+      <DraggableCellActions data={data} scopeId={scopeId} hideTopN={hideTopN}>
+        {content}
+      </DraggableCellActions>
     );
   }
-  return (
-    <DraggableOnWrapperComponent
-      dataProvider={dataProvider}
-      hideTopN={hideTopN}
-      onFilterAdded={onFilterAdded}
-      fieldType={fieldType}
-      isAggregatable={isAggregatable}
-      render={render}
-      scopeId={scopeId}
-      truncate={truncate}
-    />
-  );
-};
-
-export const DraggableWrapper = React.memo(DraggableWrapperComponent);
+);
+DraggableOnWrapper.displayName = 'DraggableOnWrapper';
+
+export const DraggableWrapper: React.FC<Props> = React.memo(
+  ({ dataProvider, isDraggable = false, render, scopeId, truncate, hideTopN }) => {
+    const data = useMemo(() => {
+      const { value, field } = dataProvider.queryMatch;
+      return { value, field };
+    }, [dataProvider.queryMatch]);
+
+    const content = useMemo(
+      () => (
+        <div tabIndex={-1} data-provider-id={getDraggableId(dataProvider.id)}>
+          {truncate ? (
+            <TruncatableText data-test-subj="render-truncatable-content">
+              {render(dataProvider, null, {
+                isDragging: false,
+                isDropAnimating: false,
+                isClone: false,
+                dropAnimation: null,
+                draggingOver: null,
+                combineWith: null,
+                combineTargetFor: null,
+                mode: null,
+              })}
+            </TruncatableText>
+          ) : (
+            <ProviderContentWrapper
+              data-test-subj={`render-content-${dataProvider.queryMatch.field}`}
+            >
+              {render(dataProvider, null, {
+                isDragging: false,
+                isDropAnimating: false,
+                isClone: false,
+                dropAnimation: null,
+                draggingOver: null,
+                combineWith: null,
+                combineTargetFor: null,
+                mode: null,
+              })}
+            </ProviderContentWrapper>
+          )}
+        </div>
+      ),
+      [dataProvider, render, truncate]
+    );
 
+    if (!isDraggable) {
+      if (disableHoverActions(scopeId)) {
+        return <>{content}</>;
+      }
+      return (
+        <DraggableCellActions data={data} scopeId={scopeId} hideTopN={hideTopN}>
```

Comment:
You are right, I named this component this way since it composes _CellActions_ for the _Draggables_, but I agree this name suggests "draggability". Everything in this directory is named with _draggable_ or _drag_and_drop_ prefix. Would it make more sense to name it `DragAndDropCellActions`?

Reply:
Right, that makes more sense. It just DraggableCellActions and DragAndDropCellActions both sounded like it's draggable. What about <DraggableDisabledCellActions>? It still sounded terrible but it's clear when comparing to `DraggableOnWrapper` 😅

Reply:
Renamed to `CellActionsWrapper` so it does not sound draggable 🙂 https://github.com/elastic/kibana/pull/182092/commits/ab1bc0ea2cdfddaeae5d4abeb618ad9585b1da39